src/main/java/com/maxmind/minfraud/response/Email.java

```diff
@@ -9,17 +9,20 @@
 public final class Email extends AbstractModel {
     private final Boolean isFree;
     private final Boolean isHighRisk;
+    private final String firstSeen;
 
     public Email(
             @JsonProperty("is_free") Boolean isFree,
-            @JsonProperty("is_high_risk") Boolean isHighRisk
+            @JsonProperty("is_high_risk") Boolean isHighRisk,
+            @JsonProperty("first_seen") String firstSeen
```

Comment:
Made a change in 8ad2b57ba.  It passes the tests, but the new test that I added passes even without the two argument constructor.